tests/Application/templates/bundles/SyliusAdminBundle/Order/Show/Summary/_item.html.twig

```diff
@@ -24,6 +25,10 @@
     </td>
     <td class="right aligned unit-price">
         {{ money.format(item.unitPrice, order.currencyCode) }}
+        + {{ money.format(item.getAdjustmentsTotalRecursively(customerOptionAdjustment), order.currencyCode) }}
+    </td>
+    <td class="right aligned unit-order-discount">
+        <span style="font-style: italic;">{{ money.format(item.units.first.adjustmentsTotal(orderPromotionAdjustment), order.currencyCode) }}</span>
```

Comment:
Double check against this PR: https://github.com/Sylius/Sylius/pull/11114 if there should be an accumulated distributed order discount

Reply:
Copied the new version over.